src/lxplib.h

```diff
@@ -2,8 +2,8 @@
 ** See Copyright Notice in license.html
 */
 
-#define LuaExpatCopyright	"Copyright (C) 2003-2007 The Kepler Project, 2013-2022 Matthew Wild"
```

Comment:
Again lets not mix housekeeping, release activities, and feature/fixes in the same commit & PR.